tests/kernel/workq/work/src/main.c

```diff
@@ -221,6 +222,21 @@ ZTEST(work, test_unstarted)
 	zassert_equal(rc, -ENODEV);
 }
 
+static void cooplo_main(void *workq_ptr, void *p2, void *p3)
+{
+	ARG_UNUSED(p2);
+	ARG_UNUSED(p3);
+
+	struct k_work_q *queue = (struct k_work_q *)workq_ptr;
+
+	struct k_work_queue_config cfg = {
+		.name = "wq.cooplo",
+		.no_yield = true,
+	};
+
+	k_work_queue_run(queue, &cfg);
+}
+
 static void test_queue_start(void)
```

Comment:
this is now not testing `k_work_queue_start` as the test name says? Maybe you should be adding a new test for `k_work_queue_run` instead?